whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -463,4 +462,40 @@ private static VCard createVCard(final RpslObject rpslObject) {
 
         return builder.build();
     }
+
+    private static void handleLanguageAttribute(final RpslObject rpslObject, final RdapObject rdapObject) {
```

Comment:
handleLanguageAttribute and handleCountryAttribute look very similar, can they be generalised?